### ISSUE TITLE
Changing inactive title buttons in customChrome.css doesn't work

### ISSUE THREAD

**Issue by quiltedstars** [vinceliuice/WhiteSur-firefox-theme#17]
I've managed to change the inactive title buttons (close/minimize/maximize) to be 'always on' (ie. never disappearing to grey) in the actual titlebutton-dark.css file, but if I try and copy the same code to my custom Chrome file, the icons disappear entirely. I've tried appending !important to the end of the url no-repeat, but it still doesn't show in my customChrome. Any insight would be much appreciated.

**Comment by AdamXweb** [comment]
TLDR; Check the background url in customChrome to point to the file.

CSS has [Specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), meaning that even when using [!important](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#the_!important_exception), the browser will still apply the most relevant CSS deceleration to an element, which in turn, determines the property value to apply to the element. ([good visual example](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#making_selectors_specific_with_and_without_adding_specificity))

Could you share the code you're using in the customChrome and your modification in the `titlebutton-dark.css`?

The likely issue is that the background URL for the icon is pointing to a file that doesn't exist. 
e.g. `background: url("../titlebuttons...` points up a folder from where that CSS file is, then into the titlebuttons folder to find the image.
<img width="684" alt="Screenshot 2023-03-30 at 3 45 20 pm" src="https://user-images.githubusercontent.com/6800453/228731931-468ce4a0-1d74-42d5-88ec-a726bd04f771.png">

Check to see where the titlebuttons folder is from your customChrome file and correctly link it.
